src/components/create/container/toolbar/toolbar.component.js

```diff
@@ -33,13 +31,19 @@ export class ToolbarComponent extends ProjectComponent {
     this.router = router;
     this.currentProjectService = currentProjectService;
     this.storage = storage.local;
+    this.toggleLeft = false;
```

Comment:
these variables are not necessary, just check against `hideSidebar`  and `hideResult`